pytorch_lightning/plugins/training_type/ddp.py

```diff
@@ -83,7 +84,7 @@ class DDPPlugin(ParallelPlugin):
     devices (e.g. GPU) per node. It is very similar to how :mod:`torch.distributed.launch` launches processes.
     """
 
-    distributed_backend = "ddp"
+    distributed_backend = DistributedType.DDP
```

Comment:
Then, the `distributed_backend` would be `DistributedType.DDP`

Reply:
That sounds misleading.

Reply:
If you are subclassing `DDPPlugin` for `CustomDDPPlugin`, and the attribute `distributed_backend` is "ddp". It doesn't sound misleading to me. You could always override the attribute, it will update the `_distrib_type`. 

> CustomDDPPlugin won't exist in DistributedType.

If you are passing Plugin instances to Trainer, we don't check for `DistributedType`.

You could also check out [Plugins registry](https://pytorch-lightning.readthedocs.io/en/latest/advanced/plugins_registry.html) to register custom plugins.